scripts/my.cnf.sample

```diff
@@ -16,6 +16,10 @@ log-error                       = YOUR_ABS_PATH/log/mysqld.log
 lc-messages-dir                 = YOUR_ABS_PATH/share/english/
 local-infile
 
+tianmu_insert_delayed=0
```

Comment:
Why do you change the default values.

Reply:
Because if insertion is delayed, queries immediately after insertion will result in undetected results, adding insertion delay is an optimization option and cannot be used as an official release setting